readme.markdown

```diff
@@ -144,6 +144,23 @@ By default, uncaught exceptions in your tests will not be intercepted, and will
 - In-process reporting with https://github.com/DavidAnson/tape-player
 - Describe blocks with https://github.com/mattriley/tape-describe
 
+# command-line flags
+
+While running tests, top-level configurations can be passed via the command line to specify desired behavior.
+
+Available configurations are listed below:
+
+## --no-only
+This is particularly useful in a CI environment where an [only test](#testonlyname-opts-cb) is not supposed to go unnoticed.
+
+By passing the `--no-only` flag, any existing [only test](#testonlyname-opts-cb) causes tests to fail.
```

Comment:
let's link to this section from the `test.only` section.

Reply:
done